usecases/cem/opev/public.go

```diff
@@ -26,23 +26,46 @@ func (e *OPEV) CurrentLimits(entity spineapi.EntityRemoteInterface) ([]float64,
 		return nil, nil, nil, err
 	}
 
+	lc, err := client.NewLoadControl(e.LocalEntity, entity)
+	if err != nil {
+		return nil, nil, nil, err
+	}
+
 	meas, err := client.NewMeasurement(e.LocalEntity, entity)
 	if err != nil {
 		return nil, nil, nil, err
 	}
 
-	filter := model.MeasurementDescriptionDataType{
-		MeasurementType: util.Ptr(model.MeasurementTypeTypeCurrent),
-		CommodityType:   util.Ptr(model.CommodityTypeTypeElectricity),
-		Unit:            util.Ptr(model.UnitOfMeasurementTypeA),
-		ScopeType:       util.Ptr(model.ScopeTypeTypeACCurrent),
+	filter := model.LoadControlLimitDescriptionDataType {
+		LimitType:     util.Ptr(model.LoadControlLimitTypeTypeMaxValueLimit),
+		LimitCategory: util.Ptr(model.LoadControlCategoryTypeObligation),
+		Unit:          util.Ptr(model.UnitOfMeasurementTypeA),
+		ScopeType:     util.Ptr(model.ScopeTypeTypeOverloadProtection),
 	}
-	measDesc, err := meas.GetDescriptionsForFilter(filter)
+	
+	limitDescs, err := lc.GetLimitDescriptionsForFilter(filter)
 	if err != nil {
 		return nil, nil, nil, err
 	}
+	if len(limitDescs) != 3 {
```

Comment:
This check is not correct, as wallboxes could be connected with 1 Phase only. So we should only check for the array not to be empty.